host/3.0/buster/amd64/java/java8/java8-composite.template

```diff
@@ -11,15 +13,23 @@ ENV AzureWebJobsScriptRoot=/home/site/wwwroot \
     DOTNET_USE_POLLING_FILE_WATCHER=true \
     HOST_VERSION=${HOST_VERSION}
 
+RUN apt-get -qq update \
+    && apt-get -qqy install curl \
+    && rm -rf /var/lib/apt/lists/*
+
+RUN mkdir -p ${JAVA_HOME} \
+    && curl -fsSL -o /tmp/jdk.tar.gz ${BASE_JDK_URL}/OpenJDK8U-jdk_x64_linux_hotspot_${JAVA_VERSION}.tar.gz \
+    && tar -xzf /tmp/jdk.tar.gz -C ${JAVA_HOME} --strip-components=1 \
+    && rm -f /tmp/jdk.tar.gz
+
 COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
 COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
-COPY --from=jre [ "/usr/lib/jvm/zre-hl-8-azure-amd64", "/usr/lib/jvm/zre-8-azure-amd64" ]
```

Comment:
This is probably just fine.  We might investigate the idea of adding an intermediate jre image.   Then we could continue using basically this directive.  
I think that this makes more sense than the current way that you are doing in order to separate responsibilities more cleanly.